packages/react/src/components/form/authenticating/index.tsx

```diff
@@ -20,9 +21,11 @@ import * as styles from "./authenticating.css";
 import { Children, useCallback, useEffect, useRef, useState } from "react";
 import { TOTPState } from "./totp";
 import { Delayed } from "@slashid/react-primitives";
-import { BASE_RETRY_DELAY_MS } from "./authenticating.constants";
 import { useInternalFormContext } from "../internal-context";
 import { AuthenticatingState } from "../flow";
+import { TIME_MS } from "../types";
+
+const DELAY_BEFORE_RETRY = TIME_MS.second * 30;
```

Comment:
Previously it was 2 seconds, now we're changing it to 30 (which I guess makes sense), just curious if this number is arbitrary or was it an outcome of some request/conversation?

Reply:
Got it!